qa/suites/kcephfs/recovery/debug/mds_client.yaml

```diff
@@ -1,12 +0,0 @@
-overrides:
```

Comment:
The containing directory 'qa/suites/kcephfs/recovery/debug` needs deleted too. Ditto for `recovery/dirfrag`.